### ISSUE TITLE
Temporal Dimension?

### ISSUE THREAD

**Issue by tamos** [uchicago-computation-workshop/james_evans#2]
In your exploratory analysis, did you explore temporal dimensions of collaboration between polarized teams?

While reading the paper, I was struck by the conclusion that polarized teams produce higher-quality results. However, I wonder if those benefits are in some way offset by the time it takes them to arrive at those results?

**Comment by rodrigovaldes** [comment]
Good point. Although, maybe Wikipedia editors like to contribute to these controversies!

**Comment by LeosonH** [comment]
this question was in my mind as well! would love to hear a response.
